### ISSUE TITLE
Create command-line interaction for chigger

### ISSUE THREAD

**Issue by aeslaughter** [idaholab/moose#12095]
## Rationale
Chigger would be more valuable if you could have some method for moving items around when 
you are creating a script.

## Description
All interaction will be done with the key-board and mouse, there will be no gui.

## Impact
Improved usability of chigger for creating visualizations.

**Comment by permcody** [comment]
This can probably be closed now?

**Comment by aeslaughter** [comment]
No, I am working on this in occasionally in evenings. There is more to be done, but I hope to have something in the coming months.
